config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users 
```

Comment:
You will have to specify the controllers, if you'd want to edit the generated layouts.
Example:
```
devise_for :users, controllers: {
  registrations: 'users/registrations',
  sessions: 'users/sessions'
}
```